src/components/Menu/MenuBar/MenuBar.vue

```diff
@@ -538,11 +530,9 @@ const menudata = computed<MenuItemData[]>(() => [
     type: "button",
     label: "ヘルプ",
     onClick: () => {
-      if (store.state.isHelpDialogOpen) closeAllDialog();
-      else {
-        closeAllDialog();
-        openHelpDialog();
-      }
+      Dialog.create({
+        component: HelpDialog,
+      });
```

Comment:
`closeAllDialog()`呼ぶままにしても良さそう！
```suggestion
      closeAllDialog();
      Dialog.create({
        component: HelpDialog,
      });
```

まあでもそもそも`closeAllDialog`関数が不要そうですねぇ！
なんであるんだろ･･･。念の為に置かれてるって感じかなぁ。
`closeAllDialog`側に`// FIXME: この関数は不要なはず`とかコメント書いても良いかも。